spec/rails_helper.rb

```diff
@@ -1,10 +1,7 @@
 require 'spec_helper'
 ENV['RAILS_ENV'] ||= 'test'
-require File.expand_path('../config/environment', __dir__)
-# Prevent database truncation if the environment is production
-if Rails.env.production?
-  abort('The Rails environment is running in production mode!')
-end
+require File.expand_path('../config/environment', __dir__) # Prevent database truncation if the environment is production
```

Comment:
This comment ended up on the wrong line, it should be on the below line. Not a big deal.